### DISCUSSION TITLE
技巧分享：使用utools在浏览器外OCR查词翻译

### DISCUSSION THREAD

**Discussion by anrgct** [crimx/ext-saladict#827] [Show and tell]
## 不安装浏览器也可以使用沙拉查词
`utools`是一个基于electron的工具软件，在`utools`插件中心安装`沙拉查词`的移植版，可以不借助浏览器使用`沙拉查词`。安装完成后，在`utools`输入框输入`沙拉查词`或者`saladic`可以打开查词面板，进行浏览器外部的查词使用，比较适合不在浏览器使用的环境。
![Jietu20200528-202117](https://user-images.githubusercontent.com/16172523/83145220-0e10cd00-a127-11ea-9e2a-8ceb449febf9.jpg)
![Jietu20200528-202510](https://user-images.githubusercontent.com/16172523/83145228-11a45400-a127-11ea-8f6e-7501bbb5537d.jpg)
安装完成后，在`utools`的输入框输入任意字符，当结果无匹配时，可以按回车进入沙拉查词快速查词，非常快捷方便。
![Jietu20200528-211112](https://user-images.githubusercontent.com/16172523/83145834-fe45b880-a127-11ea-8a28-5ac48bec8e1a.jpg)
`utools`的插件可以进行数据联动，把其他插件的数据传递到`沙拉查词`，比如安装`讯飞ocr`插件，把图片ocr识别后成文本后，点击`翻译`按钮就可以跳转到`沙拉查词`，实现OCR+翻译联动。
![Jietu20200528-202736](https://user-images.githubusercontent.com/16172523/83145235-136e1780-a127-11ea-8378-650a38fa43ad.jpg)

**Comment by crimx** [comment]
移植大大厉害 Orz

**Comment by anrgct** [comment]
> 移植大大厉害 Orz

还是大佬厉害，开发时就把模拟全写了，我捡个现成的 Orz

**Comment by 1degrees** [comment]
流弊啊 大佬

**Comment by lee88688** [comment]
现在utools商店已经搜索不到沙拉查词了

**Reply by crimx** [reply]
https://github.com/anrgct/utools-saladict
